### ISSUE TITLE
[en] typo: "Mananger" in Architecture: Node documentation

### ISSUE THREAD

**Issue by daodennis** [kubernetes/website#34208]
**Problem**
Typo in Architecture: 
Introduced in: https://github.com/kubernetes/website/pull/32406 v.124 documentation

**Proposed Solution:**
```diff
- A node shutdown action may not be detected by kubelet's Node Shutdown Mananger
```
```diff
+ A node shutdown action may not be detected by kubelet's Node Shutdown Manager
```

**Page to Update:**
https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/architecture/nodes.md
https://kubernetes.io/docs/concepts/architecture/nodes/

I looked for existing bugs related to this particular typo and didn't find any.
https://github.com/kubernetes/website/search?q=Mananger 
Curiously enough this permutation of manager exists in the one language combination it might have a chance of being correct in: French/Swiss related documentation. Although it is likely a typo but I am not a native speaker there.

**Additional Information.**
/language en
/good-first-issue
/kind documentation
/kind bug

**Comment by daodennis** [comment]
I'm unclear if I have to to be a corporate contributor or I can be an individual contributor so I am checking internally before I take this PR out of draft.

**Comment by Sea-n** [comment]
/assign @daodennis
Maybe you would like to let others know you are working on it using assign as well.

**Comment by daodennis** [comment]
The PR that fixes this bug has been merged, how what do I do? Do I just close this?

**Comment by Sea-n** [comment]
Yeah, and next time you can use `Fixes #34208` in PR description so that issue will be close automatically.

BTW, next time don't put `help wanted` and `good first issue` labels if you want to do it yourself.
